README.md

```diff
@@ -94,6 +94,13 @@ For further insights on enhancing RAG applications with dense content representa
 | --- | --- |
 [/RAG/04_advanced_redisvl.ipynb](python-recipes/RAG/04_advanced_redisvl.ipynb) | Notebook for additional tips and techniques to improve RAG quality |
 
+## Agents
```

Comment:
@bsbodden added a section to the readme for this. Feel free to add more description if you feel this is to basic!